.gitignore

```diff
@@ -3,7 +3,6 @@
 /composer.phar
 /app/cache/*
 /app/config/sculpin_kernel_*.yml
-/app/config/sculpin_site_*.yml
```

Comment:
Removed from .gitignore so future changes to `app/config/sculpin_site_prod.yml` will be shown by git.